src/pages/RegisterPage.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import firebase from 'firebase/app';
+import 'firebase/auth';
+import PageTitle from '../atoms/PageTitle';
+import TextInputRow from '../atoms/TextInputRow';
+import SubmitButton from '../atoms/SubmitButton';
+import store from '../Redux/store';
+import UserChange from '../Redux/actions/UserChange/UserChange';
+
+const RegisterPage = () => {
+  const [userInfo, setUserInfo] = React.useState({
+    Name: '', Email: '', Password: '', Error: '',
+  });
+
+  function handleNameInput(input) { setUserInfo({ ...userInfo, Name: input }); }
+  function handleEmailInput(input) { setUserInfo({ ...userInfo, Email: input }); }
+  function handlePasswordInput(input) { setUserInfo({ ...userInfo, Password: input }); }
+
+  async function newUserSubmission(event) {
+    event.preventDefault();
+    await firebase.auth().createUserWithEmailAndPassword(userInfo.Email, userInfo.Password)
```

Comment:
You used `await` but didn't really leverage it anywhere else here (seeing lots of `then` statements).